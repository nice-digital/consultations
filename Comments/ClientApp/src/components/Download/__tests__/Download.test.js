import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import DownloadWithRouter from "../Download";

const fakeProps = {
	match: {
		url: "/1/review",
		params: {
			consultationId: 1,
		},
	},
	location: {
		pathname: "/1/review",
		search: "?sourceURI=consultations%3A%2F%2F.%2Fconsultation%2F1%2Fdocument%2F1",
	},
	history:{
		location:{
			search: "",
		},
		listen: function(){},
	},
	basename: "/consultations",
};

test("should match snapshot with supplied data", () => {
	window.__PRELOADED__ = { isAuthorised: true};
	const {container} = render(
		<MemoryRouter>
			<DownloadWithRouter {...fakeProps} />
		</MemoryRouter>,
	);
	expect(container).toMatchSnapshot();
});
