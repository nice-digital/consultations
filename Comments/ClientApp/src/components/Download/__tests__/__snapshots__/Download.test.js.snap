// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with supplied data 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;
